source/_integrations/homee.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: Homee
+description: Instructions on how to setup Homee devices in Home Assistant.
+ha_category:
+  - Cover
+ha_config_flow: true
+ha_release: 2025.2
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@Taraman17'
+ha_domain: homee
+ha_platforms:
+  - cover
+ha_integration_type: hub
+---
+
+[Homee](https://hom.ee) is a smart home system, able to integrate various protocols such as Z-Wave, Zigbee, EnOcean, and more. The Homee {% term integration %} will let you use the {% term device devices %} from your Homee in Home Assistant.
+
+There is currently support for the following device types in Home Assistant:
+
+- cover
+- sensor
+
+More {% term platform platforms %} will follow soon, as integration moves to core step by step.
+
+## Prerequisites
+
+You need to create a new user in your Homee that is only used by Home Assistant. For security reasons, it's recommended to:
+
+- Use a strong, unique password.
+- Set the user's permissions for what you what to do from Home Assistant.
+- Do not use this account for any other purpose.
+
+1. In your Homee app tap the menu button in the top-left corner.
+2. Tap the gear wheel icon on the top right of the opening menu.
```

Comment:
```suggestion
2. On the top right of the opening menu, tap the gear {% icon "mdi:gear-outline" %} icon.
```
Location before action. 
Gear: we have an icon for this :tada: , in case it matches what you have in your software.